src/services/lesson/index.js

```diff
@@ -66,7 +66,7 @@ class LessonCopyService {
 
 					let topic = res;
 
-					return FileModel.find({path: {$regex: sourceLesson.courseId._id}}).then(files => {
+					return FileModel.find({path: {$regex: sourceLesson.courseId._id.toString() }}).then(files => {
```

Comment:
This is old Filemodel-Code.

Reply:
that is from the latest master branch.
https://github.com/schul-cloud/schulcloud-server/blob/master/src/services/lesson/index.js#L69

It currently throws an error as $regex expects a string.